src/Components/CardList.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+
+import Card from './Card';
+import { getProductsFromCategoryAndQuery } from '../services/api';
+
+class CardList extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      results: [],
+    };
+
+    this.getList = this.getList.bind(this);
+  }
+
+  componentDidMount() {
+    this.getList();
+  }
+
+  async getList() {
+    const { results } = await getProductsFromCategoryAndQuery('MLB1384', 'Cadeira');
+    this.setState({ results });
+  }
```

Comment:
Por Padrão estamos utilizando arrowfunction em nossas funções, adicionei ela na sugestão abaixo, da uma olhada 
:+1: 
```suggestion
   getList = async () => {
    const { results } = await getProductsFromCategoryAndQuery('MLB1384', 'Cadeira');
    this.setState({ results });
  }
```